introduction.tex

```diff
@@ -317,92 +334,108 @@ \subsection{Capabilities required for modeling transition scenarios}
 recycle facilities, reactor facilities, back-end features, starting
 the new fuel cycle, materials queuing and prioritization under
 capacity limitations, and energy demand algorithms. Brown et al.
-categorize each functionality into three tiers - basic, integral,
-and exemplary. The functionalities, features, and their hierarchies
+categorize each feature into three tiers - basic, integral,
+and exemplary. Basic features are essential in modeling a fuel
+cycle, such as conservation of mass in composition changes,
+facility deployment and retirement, and modeling of front-end
+facilities like an enrichment facility.
+Integral features are
+needed to bind the basic functionalities to process
+transition scenarios. Examples of integral features are
+demand-driven deployment of fuel cycle facilities and
+material prioritization.
+Finally, Exemplary features enable exploration
```

Comment:
Should Exemplary be capitalized? Maybe "Basic" "Integral" and "Exemplary" should all be italicized or bold, to make clear that these are jargon defined by brown et al.